app/tests/spec/views/mixins/connect-another-device-mixin.js

```diff
@@ -0,0 +1,405 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const { assert } = require('chai');
+  const BaseView = require('views/base');
+  const ConnectAnotherDeviceMixin = require('views/mixins/connect-another-device-mixin');
+  const Constants = require('lib/constants');
+  const ExperimentMixin = require('views/mixins/experiment-mixin');
+  const Cocktail = require('cocktail');
+  const p = require('lib/promise');
+  const Relier = require('models/reliers/relier');
+  const sinon = require('sinon');
+  const Template = require('stache!templates/test_template');
+  const User = require('models/user');
+  const UserAgentMixin = require('views/mixins/user-agent-mixin');
+  const { createRandomHexString } = require('../../../lib/helpers');
+  const VerificationReasonMixin = require('views/mixins/verification-reason-mixin');
+
+  const VALID_UID = createRandomHexString(Constants.UID_LENGTH);
+
+  var View = BaseView.extend({
+    template: Template
+  });
+
+  Cocktail.mixin(
+    View,
+    ConnectAnotherDeviceMixin,
+    ExperimentMixin,
```

Comment:
The fact that a view needs so many mixins to use the ConnectAnotherDeviceMixin is a serious smell.